include/sound/sof/ext_manifest4.h

```diff
@@ -90,6 +90,18 @@ struct sof_man4_segment_desc {
 	uint32_t file_offset;
 } __packed;
 
+/* bit definition for type in sof_man4_module */
+#define SOF_IPC4_MODULE_LOAD_TYPE_MASK 0xf
+#define SOF_IPC4_MODULE_LOAD_TYPE_SHIFT 0
+#define SOF_IPC4_MODULE_AUTO_START_MASK 0x10
+#define SOF_IPC4_MODULE_AUTO_START_SHIFT 4
```

Comment:
`0x10 << 4` == `1 << 8`, bit3...bit7 area is unused?

Reply:
0x10 >> 4